### ISSUE TITLE
what does it mean - Plotting two features?

### ISSUE THREAD

**Issue by rojuvalu** [broadinstitute/wot#37]
Hi,
I'm really sure if this is correct to be posted here, but I'll give it a try anyway.

I'm trying to understand the algorithm and work my way through the examples. In the first example by generating the data you offer a method to visualise the data, but the script offers to plot only two features. What does it mean? 
What two features of the data are plotted here? Does it means two columns of the matrix.txt data set?

Is there a way to plot more than just these two features?
thanks

**Comment by joshua-gould** [comment]
I updated the example to plot a 2d embedding of the data. You can color the
embedding as you wish (e.g. by the expression of a gene, cluster labels,
etc.)

**Comment by rojuvalu** [comment]
thanks. this is great and fast work

**Comment by frymor** [comment]
Hi, what happened to the script `02_plotting_two_features.py`?
it disappeared from the repository and running it now causes an error
```
$ python 02_plotting_two_features.py 
Traceback (most recent call last):
  File "02_plotting_two_features.py", line 18, in <module>
    wot.io.add_row_metadata_to_dataset(ds, days_path=days_file)
TypeError: add_row_metadata_to_dataset() got an unexpected keyword argument 'days_path'
```

**Comment by geoffschieb** [comment]
We are in the process of replacing the Examples page with a Tutorial page.
This will be available next week.

Although if you look carefully on Github you will see that the notebooks
are already there! They're just not quite complete now.

Best,
Geoff

**Comment by frymor** [comment]
It will be great if you can let me k wo, when the new scripts/ new version is online.

**Comment by geoffschieb** [comment]
Hi rojuvalu,

We have just released WOT 1.0 and a tutorial:
https://broadinstitute.github.io/wot/tutorial/

Best,
Geoff
